wire-ios/Wire-iOS/Sources/UserInterface/DeviceView/Views/DeviceDetailsView.swift

```diff
@@ -64,7 +64,7 @@ struct DeviceDetailsView: View {
 
     var mlsView: some View {
         VStack(alignment: .leading) {
-            sectionTitleView(title: L10n.Localizable.Device.Details.Secion.Mls.signature.uppercased())
+            sectionTitleView(title: L10n.Localizable.Device.Details.Section.Mls.signature.uppercased())
```

Comment:
suggestion: Maybe here we need to use .localizedUppercased to make sure the end result is localized.